notebook/static/base/js/utils.js

```diff
@@ -879,4 +890,4 @@ define([
     };
 
     return utils;
-}); 
+}); 
```

Comment:
Does your editor strip a final newline? Mine always adds them. Might get annoying if we work on the same files...